yarn-completion.plugin.zsh

```diff
@@ -131,8 +129,13 @@ _yc_yarn_commands_completion() {
       'team[Maintain team memberships]' \
       'unlink[Unlink a previously created symlink for a package.]' \
       'upgrade[Upgrades packages to their latest version based on the specified range.]' \
+      'upgrade --latest[Upgrades packages to their latest.]' \
```

Comment:
This is only about subcommands, not their options. However, if you want to add option completion that would be a nice idea! In the meantime I don't wanna get messy with some options printed here.
```suggestion
```